packages/stylelint-config-airbnb/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "airbnb"
+}
```

Comment:
Done in https://github.com/airbnb/css/pull/23/commits/ff18d84c4b306ecb6db23d3957337c16b5b9591b